FlightAppInSwiftUI/View/RoundTripView.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  RoundTripView.swift
+//  FlightAppInSwiftUI
+//
+//  Created by Dagmawit Alemayehu on 15/03/2023.
+//
+
+import SwiftUI
+
+struct RoundTripView: View {
+    @State private var fromRoundTripText: String = String()
+    @State private var toRoundTripText: String = String()
+    @State private var departureDate = Date.now
+    @State private var returnDate = Date.now
+    var body: some View {
+        ScrollView {
+            VStack (spacing: 30){
+                ZStack {
```

Comment:
from this line to 27th line, you could notice that there is a repetitive code. Could you please refactor it? Maybe you could create a opaque type view returning function that takes all the same components and modifiers applied repetitively in the same area.

Reply:
Done 👍
I've removed the repetitive modifier code and put them in a new swift file that contains functions that return opaque type views